techniques/html/H99.html

```diff
@@ -35,8 +35,8 @@ <h2>Description</h2>
 			<h2>Examples</h2>
     <section class="example">
 				<h3>Page list in <abbr title="HyperText Markup Language">HTML</abbr> publications</h3>
-				<p>To add a page list to a web-based document, the role <code class="language-xml">"doc-pagelist"</code> is added to the element (typically a <code class="language-html">nav</code> element) that encloses the list and its title.</p>
-				<pre xml:space="preserve"><code class="language-html">&lt;nav role="doc-pagelist" aria-labelledby="pglist"&gt;
+				<p>To add a page list to a web-based document, the role <code>"doc-pagelist"</code> is added to the element (typically a <code>nav</code> element) that encloses the list and its title.</p>
```

Comment:
```suggestion
				<p>To add a page list to a web-based document, <code>role="doc-pagelist"</code> is added to the element (typically a <code>nav</code> element) that encloses the list and its title.</p>
```

and could then benefit from highlighting